Kitodo/src/main/java/de/sub/goobi/forms/MassImportForm.java

```diff
@@ -344,10 +341,10 @@ public void uploadFile() throws IOException {
             basename = basename.substring(1);
         }
         if (basename.contains("/")) {
-            basename = basename.substring(basename.lastIndexOf("/") + 1);
+            basename = basename.substring(basename.lastIndexOf('/') + 1);
         }
         if (basename.contains("\\")) {
-            basename = basename.substring(basename.lastIndexOf("\\") + 1);
+            basename = basename.substring(basename.lastIndexOf('\\') + 1);
```

Comment:
Are you sure about this change? `lastIndexOf` can be called with a string type and int type value and in this case the string value must be used as we are checking for double backslash characters. Or did I missed something?

Reply:
It is escape character, so this change works correctly.

Reply:
Thank you for this explanation - escaping characters confuse me sometimes.